src/osgWrappers/serializers/osg/Texture.cpp

```diff
@@ -244,13 +244,13 @@ REGISTER_OBJECT_WRAPPER( Texture,
 
     ADD_FLOAT_SERIALIZER( ShadowAmbient, 0.0f );  // _shadow_ambient
 
-    {
+   /* {
```

Comment:
We can't remove elements from the serializers without versioning their removal, otherwise we'd break backwards compatibility.